main.go

```diff
@@ -83,42 +83,56 @@ func main() {
 		log.Log.Infow("Initialized connection to Redis", "i", i)
 
 		aggregatedRedisClients[i] = redisClients
+		clientsSize := len(redisClients)
+
+		redisPubsAggregationEntry := make([]chan<- *redispub.Publication, clientsSize)
+		stopRedisPubsEntry := make([]chan bool, clientsSize)
+
+		for j := 0; j < clientsSize; j++ {
+			redisClient := redisClients[i]
+
+			redisPubs := make(chan *redispub.Publication, bufferSize)
+			redisPubsAggregationEntry[j] = redisPubs
+
+			stopRedisPub := make(chan bool)
+			stopRedisPubsEntry[j] = stopRedisPub
+
+			waitGroup.Add(1)
+
+			// We crate two goroutines:
```

Comment:
create

Reply:
lol that was there, i'll do a spelling pass